metrics-micrometer/src/main/java/org/axonframework/micrometer/GlobalMetricRegistry.java

```diff
@@ -148,30 +239,78 @@ public Configurer registerWithConfigurer(Configurer configurer) {
         return new MultiMessageMonitor<>(Arrays.asList(messageCountingMonitor, messageTimerMonitor));
     }
 
+    /**
+     * Registers new metrics to the registry to monitor an {@link EventBus}. The monitor will be registered with the
```

Comment:
Nit: might be nice to deviate a little with the documentation compared to the `registerEventBus(String)` method, by for example doing the following:

```suggestion
     * Registers new metrics to the registry to monitor an {@link EventBus} using {@link Tag}s through the given {@code tagsBuilder}. The monitor will be registered with the
```

Reply:
I'd expect a similar adjustment to the other registration methods which add the `tagsBuilder` parameter.